numpy/lib/function_base.py

```diff
@@ -3562,14 +3562,16 @@ def sinc(x):
     r"""
     Return the normalized sinc function.
 
-    The sinc function is :math:`\sin(\pi x)/(\pi x)`.
+    The sinc function is equal to :math:`\sin(\pi x)/(\pi x)` for any argument
+    :math:`x\ne 0`. ``sinc(0)`` takes the limit value 1, making ``sinc`` not
+    only everywhere continuous but also analytic.
```

Comment:
```suggestion
    only everywhere continuous but also infinitely differentiable.
```

My one nit is that our audience is pretty broad and might not know what an analytic function is and why that's important, but they do know about differentiability and why the behavior around 0 (given the formula) might be a problem for that. The popularity of ML and data science brings in a lot of people relatively early in their math education.

I'm not too fussed about it either way, though.

Reply:
Thank you for the very quick reply! I agree with the suggested change and the reasoning behind it. Unfortunately I am not sure how to accept it from the GitHub interface, or even if I have to take any step at all in that regard.